<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1454</width>
    <height>1168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="pushButton_StartGame">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>START</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
         <widget class="QWidget" name="widget_GoBoard" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>900</width>
            <height>900</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>900</width>
            <height>900</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(200, 184, 135)</string>
          </property>
          <widget class="QGraphicsView" name="graphicsView_GoBoard">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>900</width>
             <height>900</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>900</width>
             <height>900</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>900</width>
             <height>900</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(200, 184, 135)</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>1073</height>
         </size>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>110</y>
           <width>501</width>
           <height>261</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item alignment="Qt::AlignHCenter">
           <widget class="QWidget" name="widget_2" native="true">
            <property name="minimumSize">
             <size>
              <width>242</width>
              <height>209</height>
             </size>
            </property>
            <widget class="QWidget" name="verticalWidget" native="true">
             <property name="geometry">
              <rect>
               <x>9</x>
               <y>0</y>
               <width>231</width>
               <height>261</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="sizeConstraint">
               <enum>QLayout::SetMinimumSize</enum>
              </property>
              <item alignment="Qt::AlignHCenter|Qt::AlignTop">
               <widget class="QLineEdit" name="lineEdit_2">
                <property name="font">
                 <font>
                  <family>MS Shell Dlg 2</family>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="cursor">
                 <cursorShape>IBeamCursor</cursorShape>
                </property>
                <property name="styleSheet">
                 <string notr="true">
background-color: rgba(255, 255, 255, 0);</string>
                </property>
                <property name="text">
                 <string>BLACK</string>
                </property>
                <property name="frame">
                 <bool>false</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignHCenter|Qt::AlignTop">
               <widget class="QLabel" name="label_IconB">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
               <widget class="QLineEdit" name="BlackPlayerName">
                <property name="minimumSize">
                 <size>
                  <width>114</width>
                  <height>19</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string>
background-color: rgba(255, 255, 255, 0);</string>
                </property>
                <property name="frame">
                 <bool>false</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="placeholderText">
                 <string>Black Player</string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
               <widget class="QLineEdit" name="lineEdit_BlackCaptures">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QWidget" name="widget_3" native="true">
            <property name="minimumSize">
             <size>
              <width>241</width>
              <height>209</height>
             </size>
            </property>
            <widget class="QWidget" name="verticalWidget_2" native="true">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>241</width>
               <height>261</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="sizeConstraint">
               <enum>QLayout::SetMinimumSize</enum>
              </property>
              <item alignment="Qt::AlignHCenter|Qt::AlignTop">
               <widget class="QLineEdit" name="lineEdit_4">
                <property name="font">
                 <font>
                  <family>MS Shell Dlg 2</family>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="cursor">
                 <cursorShape>IBeamCursor</cursorShape>
                </property>
                <property name="styleSheet">
                 <string notr="true">
background-color: rgba(255, 255, 255, 0);</string>
                </property>
                <property name="text">
                 <string>WHITE</string>
                </property>
                <property name="frame">
                 <bool>false</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignHCenter|Qt::AlignTop">
               <widget class="QLabel" name="label_IconW">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
               <widget class="QLineEdit" name="WhitePlayerName">
                <property name="minimumSize">
                 <size>
                  <width>114</width>
                  <height>19</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string>
background-color: rgba(255, 255, 255, 0);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="frame">
                 <bool>false</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="placeholderText">
                 <string>White Player</string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
               <widget class="QLineEdit" name="lineEdit_WhiteCaptures">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>390</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>PASS</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>390</y>
           <width>75</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>RESIGN</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_SaveGame">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Save Game</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_Exit">
         <property name="geometry">
          <rect>
           <x>410</x>
           <y>10</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="plainTextEdit_Probs">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>430</y>
           <width>501</width>
           <height>300</height>
          </rect>
         </property>
         <property name="plainText">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1454</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
